ROMFS/px4fmu_common/init.d/airframes/13003_quad_tailsitter

```diff
@@ -38,7 +38,6 @@ param set-default PWM_MAIN_MAX 2000
 param set-default VT_MOT_ID 1234
 param set-default VT_IDLE_PWM_MC  1080
 param set-default VT_TYPE 0
-set MAV_TYPE 20
```

Comment:
maybe to add `param set-default MAV_TYPE 20 ` for tail sitters, otherwise MAV_TPYE is 22